frontends/common/parser_options.cpp

```diff
@@ -483,7 +483,10 @@ void ParserOptions::dumpPass(const char *manager, unsigned seq, const char *pass
             exit(1);
         }
         if (match) {
-            cstring suffix = cstring("-") + name;
+            char buf[8];
+            static int uid;
```

Comment:
Can you please make this a member variable of `ParserOptions` rather than a static one? I think that starting dump count from 0 for new instance of options makes sense.

Reply:
Agree, we use `static` a lot and is causing quite a few problems these days.

Reply:
Needs to be a mutable member, as this is a const function, but trivially done.

Reply:
I did not notice the const... it is not the nicest use of mutable, but I prefer it over `static`.